docs-chef-io/content/inspec/cli.md

```diff
@@ -38,7 +38,7 @@ This subcommand has the following additional options:
 * `-o`, `--output=OUTPUT`
     Save the archive to a path.
 * `--overwrite`, `--no-overwrite`
-    Overwrite existing archive.
+    Overwrite existing archives.
```

Comment:
Maybe we should revert this back to `archive` only? Since we only generate a tar for a profile.

Reply:
ok